CRM/Activity/Selector/Search.php

```diff
@@ -248,6 +248,8 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
       foreach (self::$_properties as $property) {
         if (isset($result->$property)) {
           $row[$property] = $result->$property;
+        } else {
```

Comment:
@braders our style guide (for better or worse) says that `else` must go on a new line. I'm not a personal fan of that rule, but at least we're being consistent.

Reply:
Doh, yes. I've repushed, so hopefully the next build will pass.